### PR TITLE
Updated: Joystick single tool in tone with Gmod 13

### PR DESCRIPTION
**Pull request by dvdvideo1234**
Fixed: Presets, DrawHUD unreadable and slow, Entity ownership, Server limits, Tool information

### CODE REVIEW

File: addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick.lua [hunk 1]
```diff
@@ -1,344 +1,412 @@
...
+  local clBlue  = Color(   0,   0, 255, 255 )
+  local clCyan  = Color(   0, 255, 255, 255 )
+  local clMagen = Color( 255,   0, 255, 255 )
+  local clRed   = Color( 255,   0,   0, 255 )
+  local clYello = Color( 255, 255,   0, 255 )
+  local clGreen = Color(   0, 255,   0, 255 )
+  local clInBnd = Color( 255, 165,   0, 255 )
+  local clInAct = Color(  32, 178, 170, 255 )
+
+  function TOOL:DrawToolScreen(w,h)
```
Comment (dvdvideo1234) [inline addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick.lua:297]: Improved readability of the tool screen. When drawing a single bind, more space can be utilized

File: addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick.lua [hunk 2]
```diff
@@ -1,344 +1,412 @@
...
-TOOL.Name			= "Joystick"
-TOOL.Command		= nil
-TOOL.ConfigName		= ""
+local gsToolModeOP = TOOL.Mode
+local gsToolPrefix = gsToolModeOP.."_"
+local gsToolLimits = gsToolModeOP:gsub("_multi", "").."s"
+local gsSentClasMK = "gmod_"..gsToolModeOP
+local MappingFxUID = "abcdefghijklmnopqrstuvwxyz1234567890ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+TOOL.Tab        = gsToolModeOP:match("%w+"):gsub("^%l", string.upper)
```
Comment (MattJeanes) [inline addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick.lua:7]: Please leave this at "Wire" and the below as "Joystick" as this string manipulation is unnecessarily complicated

File: addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick.lua [hunk 3]
```diff
@@ -1,344 +1,412 @@
...
+  language.Add( "tool."..gsToolModeOP..".lcontr_con"     , "Control configuration:" )
+  language.Add( "tool."..gsToolModeOP..".maxon"          , "Maximum output value when analogue or ON value when digital" )
+  language.Add( "tool."..gsToolModeOP..".maxon_con"      , "Maximum / On")
+  language.Add( "tool."..gsToolModeOP..".minoff"         , "Minimum output value when analogue or OFF value when digital" )
+  language.Add( "tool."..gsToolModeOP..".minoff_con"     , "Minimum / Off" )
+  language.Add( "tool."..gsToolModeOP..".analog"         , "Enable this when your source is analogue input" )
+  language.Add( "tool."..gsToolModeOP..".analog_con"     , "Analog input" )
+  language.Add( "tool."..gsToolModeOP..".config"         , "Click this button to open the joystick configuration. You can also right click on the world" )
+  language.Add( "tool."..gsToolModeOP..".config_con"     , "Joystick Configuration" )
+  language.Add( "undone_"..gsToolModeOP                  , "Undone Wire Joystick !" )
```
Comment (MattJeanes) [inline addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick.lua:44]: Why is there a space between the word and exclamation point?

**Review by MattJeanes — CHANGES_REQUESTED** [review]

File: addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick.lua (hunk 3 above)
Comment (dvdvideo1234) [reply addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick.lua]: Revered in 97acdb1cc9315998cdd73092fceeb1dfbf1f8afc

File: addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick.lua (hunk 2 above)
Comment (dvdvideo1234) [reply addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick.lua]: Reverted in 97acdb1cc9315998cdd73092fceeb1dfbf1f8afc

**Review by MattJeanes — APPROVED** [review]
Good stuff, thank you very much

**Pull request merged**